subcommittees/mentoring/roles/mentees.md

```diff
@@ -0,0 +1,17 @@
+# Mentees
+
+## Responsibilities
+
+
+## Time commitment
+
+
+## Requirements
+
```

Comment:
Be either a newly certified Software or Data Carpentry instructor or be in the process of completing certification.    Generally Mentees will have taught one or no workshops, but may have been helpers before.